cms/djangoapps/api/docs/decisions/001-course-apps-api.rst

```diff
@@ -0,0 +1,160 @@
+Course Apps API
+_______________
+
+Status
+======
+Proposal
+
+Context
+=======
+
+The new `Course Authoring MFE`_ includes a new UX called "Pages and Resources"
```

Comment:
Just refactored a little so that its easier to understand. 
```
The new _Course Authoring MFE_ includes a new UX called "Pages and Resources" for configuring course "apps" such as progress, wiki, teams, discussions, etc. all from one place. Currently, these apps are intermingled with the core platform and are inseparable from it, however, future apps could be implemented as plugins.

This ADR proposes a way to configure these apps from the backend that MFEs can use and potentially enable/disable these apps using API.
```